Sources/PT.PM.Matching/MatchUtils.cs

```diff
@@ -15,14 +15,14 @@ internal static List<TextSpan> AlignTextSpans(List<TextSpan> foldedTextSpans, Li
         {
             List<TextSpan> result = new List<TextSpan>(matchesLocations.Count);
 
-            int startOffset = foldedTextSpans[0].Start;
+            int startOffset = foldedTextSpans.FirstOrDefault().Start;
```

Comment:
What's the point of using `FirstOrDefault` here? It's `TextSpan.Zero` here because of `TextSpan` is a struct.